back/app/services/input_json_schema_generator_service.rb

```diff
@@ -54,4 +54,15 @@ def visit_topic_ids(field)
       end
     }
   end
+
+  def visit_cosponsor_ids(field)
```

Comment:
For now, we are not limiting the minimum number of co-sponsors via the custom field configuration.